Jenkinsfile.rdgo

```diff
@@ -63,6 +64,14 @@ node(env.NODE) {
                         ${rdgo}/ ${env.ARTIFACT_SERVER}:${rdgo}
                 """
             }
+            withCredentials([[$class: 'AmazonWebServicesCredentialsBinding',
+                              credentialsId: env['AWS_CREDENTIALS'],
+                              accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                              secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                sh """
+                    aws s3 sync ${rdgo}/build/ s3://aos-ci/rhcos/rdgo
```

Comment:
I notice we aren't using `--delete`...that's probably OK short term.  But I feel like we should set up something like Pulp or whatever to push to - that also applies to origin RPMs.

Reply:
Good point, the CLI supports `--delete`, so added in :arrow_down: